main.go

```diff
@@ -191,16 +142,17 @@ func main() {
 	router.Handle("/{uri:.*}", routerProxy)
 
 	log.Debug("Starting server", log.Data{
-		"bind_addr":           bindAddr,
-		"frontend_router_url": routerURL,
-		"zebedee_url":         zebedeeURL,
-		"recipe_api_url":      recipeAPIURL,
-		"import_api_url":      importAPIURL,
-		"dataset_api_url":     datasetAPIURL,
-		"enable_new_app":      enableNewApp,
+		"bind_addr":           cfg.BindAddr,
+		"frontend_router_url": cfg.RouterURL,
+		"zebedee_url":         cfg.ZebedeeURL,
+		"recipe_api_url":      cfg.RecipeAPIURL,
+		"import_api_url":      cfg.ImportAPIURL,
+		"dataset_api_url":     cfg.DatasetAPIURL,
+		"enable_new_app":      cfg.EnableNewApp,
+		"encryption_disabled": cfg.EncryptionDisabled,
```

Comment:
if you set the json tags mentioned ^ then you wont have to explicitly declare which fields to log out here, and log.Data can just contain the config var